src/main/java/com/icthh/xm/ms/scheduler/manager/SchedulingManager.java

```diff
@@ -241,6 +249,10 @@ private ScheduledFuture schedule(RunnableTask expirable) {
             case CRON:
                 future = taskScheduler.schedule(expirable, new CronTrigger(task.getCronExpression()));
                 break;
+            case ONE_TIME:
+                task.setEndDate(task.getStartDate().plusSeconds(task.getTtl()));
```

Comment:
ttl is not mandatory field
please handle situation if ttl is null

Reply:
+